### ISSUE TITLE
[Safari] Extension tab button focus indicator not visible on safari

### ISSUE THREAD

**Issue by riknoll** [microsoft/pxt-microbit#7017]
the focus indicator for the extension categories doesn't show up in beta in safari. for some reason the outline isn't visible

fine in chrome
